bcipy/helpers/README.md

```diff
@@ -0,0 +1,140 @@
+- `save`: methods for saving BciPy data in supported formats. For saving of triggers, see triggers.py
+- `session`: methods for managing and parsing session.json data
+- `stimuli`: methods for generating stimuli and inquiries for presentation
+- `system_utils`: utilities for extracting git version, system information and handling of logging
+- `task`: common task methods and utilities, including Trial and InquiryReshaper.
+- `triggers`: methods and data classes defining BciPy internal triggering
+- `validate`: methods for validating experiments and fields 
+- `visualization`: method for visualizing EEG data collected in BciPy. Used in offline_analysis.py.
+
+
+## Triggers
+
+Triggers consist of a label, type and a timestamp. These are what is used to align events both in real-time and after a session for further processing or model training. During operation of a BciPy task, Triggers are generated when display or other events of interest occur. These are timestamped using a monotonic clock (See helpers/clock.py).
+
+A new Trigger may defined as follows:
+```python
+from bcipy.helpers.triggers import Trigger, TriggerType
+
+# label can be any utf-8 complaint string
+nontarget_trigger = Trigger('nontarget_label', TriggerType.NONTARGET, 1.0111)
+```
+
+### Trigger Types
+
+The supported internal trigger types are as follows:
+
+```python
+NONTARGET = "nontarget"
+TARGET = "target"
+FIXATION = "fixation"
+PROMPT = "prompt"
+SYSTEM = "system"
+OFFSET = "offset"
+EVENT = "event"
+PREVIEW = "preview"
+```
+
+### TriggerHandler
+
+You can use the BciPy TriggerHandler to read and write any Triggers! 
+
+### Writing
+
+You will need three pieces of information to create a new handler for writing: 
+
+1. path where trigger data should be written
+2. name of the trigger file without extension
+3. a defined FlushFrequency. This sets how often the handler should write the data. Incrementally (FlushFrequency.EVERY) or at session end (FlushFrequency.END).
+
+```python
+from bcipy.helpers.triggers import TriggerHandler, FlushFrequency
+
+path_to_trigger_save_location = '.'
+trigger_file_name = 'triggers' # BciPy will add the correct extension. Currently, .txt is used.
+flush = FlushFrequency.END
+
+handler = TriggerHandler(path_to_trigger_save_location, trigger_file_name, flush)
+
+triggers = [
+    Trigger('test_trigger', TriggerType.SYSTEM, 1.0111),
+    Trigger('target', TriggerType.TARGET, 2),
+    Trigger('nontarget', TriggerType.NONTARGET, 3),
+]
+
+handler.add_triggers(triggers)
+handler.close() # this will call write one final time on any triggers added since last flush
+```
+
+
+### Loading
+
+To load a BciPy triggers.txt file, the TriggerHandler load method can be used. Because it is a staticmethod, the class does not need to be initialized before the load method is used. 
+
+```python
+from bcipy.helpers.triggers import TriggerHandler
+
+path_to_trigger_save_location = './triggers.txt'
```

Comment:
Is there a reason why `.txt` suffix is included during load, but not during save?

Reply:
No reason. It should be changed though. I have a ticket in the backlog to remove extensions from our other file types (session, raw data) and will add this to it! https://www.pivotaltracker.com/story/show/181115524